File: src/main/java/com/telegram/folobot/repos/FoloPidorRepo.java
package com.telegram.folobot.repos;

import com.telegram.folobot.domain.FoloPidorEntity;
import com.telegram.folobot.domain.FoloPidorId;
import org.springframework.data.domain.Sort;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface FoloPidorRepo extends CrudRepository<FoloPidorEntity, FoloPidorId> {
    List<FoloPidorEntity> findAll(Sort sort);

    List<FoloPidorEntity> findByIdChatId(Long chatId);

    List<FoloPidorEntity> findByIdChatId(Long chatId, Sort sort);
}

File: src/main/java/com/telegram/folobot/repos/FoloPidorRepo.kt
package com.telegram.folobot.repos

import com.telegram.folobot.domain.FoloPidorEntity
import com.telegram.folobot.domain.FoloPidorId
import org.springframework.data.domain.Sort
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface FoloPidorRepo : CrudRepository<FoloPidorEntity, FoloPidorId> {
    fun findAll(sort: Sort): List<FoloPidorEntity>
    fun findByIdChatId(chatId: Long): List<FoloPidorEntity>
    fun findByIdChatId(chatId: Long, sort: Sort): List<FoloPidorEntity>
}